server.js

```diff
@@ -6,6 +6,7 @@ require('dotenv').config();
 
 // Import the routes
 const index = require('./routes/index');
+var authorize = require('./routes/authorize');
 
 const app = express();
 const port = process.env.PORT;
```

Comment:
It wouldn't let me comment on the line so I am putting it here. On line 17 and 18, the `body-parser` package is deprecated. Remove it and just use `express.json()` and `express.urlencoded({ extended: false })`.